etl/steps/data/garden/who/2024-04-08/polio.py

```diff
@@ -0,0 +1,326 @@
+    #
+    # Load inputs.
+    #
+    # Load meadow acute flaccid paralysis dataset.
+    ds_meadow = paths.load_dataset("polio_afp")
+    # Load historical polio dataset
+    ds_historical = paths.load_dataset("polio_historical")
+    # Load population data to calculate cases per million population
+    ds_population = paths.load_dataset("population")
+    tb_population = ds_population["population"].reset_index()
+    # Load fasttrack Global Polio Eradication Initiative on circulating vaccine derived polio cases
+    snap_cvdpv = paths.load_snapshot("gpei.csv")
+    tb_cvdpv = snap_cvdpv.read()
+    # Dropping this as the total_cvdpv is also in the polio_afp table and has more historical data
+    tb_cvdpv = tb_cvdpv.drop(columns=["total_cvdpv"])
+    # Load regions dataset.
+    ds_regions = paths.load_dataset("regions")
+    tb_regions = ds_regions["regions"].reset_index()
+    # Load income groups dataset.
+    ds_income_groups = paths.load_dataset("income_groups")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["polio_afp"].reset_index()
+    tb_hist = ds_historical["polio_historical"].reset_index()
+    tb_hist = tb_hist.rename(columns={"cases": "total_cases"})
+    # Only need this for data prior to 2001
+    tb_hist = tb_hist[tb_hist["year"] < 2001]
+
+    # Remove data from before 2001.
+    tb = remove_pre_2001_data(tb)
+    # Remove values > 100% for "Adequate stool collection".
+    tb = clean_adequate_stool_collection(tb)
+    # Add total cases
+    tb["total_cases"] = tb["wild_poliovirus_cases"] + tb["cvdpv_cases"]
+    # Need to deal with overlapping years
+    tb = pr.concat([tb_hist, tb], axis=0)
+    tb = harmonize_countries(
+        df=tb, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
+    )
+    tb = tb.merge(tb_cvdpv, on=["country", "year"], how="left")
+    # Add region aggregates.
+    tb_reg = add_regions_to_table(
+        tb[
+            [
+                "country",
+                "year",
+                "afp_cases",
+                "wild_poliovirus_cases",
+                "cvdpv_cases",
+                "total_cases",
+                "cvdpv1",
+                "cvdpv2",
+                "cvdpv3",
+            ]
+        ],
+        regions=REGIONS,
+        ds_regions=ds_regions,
+        ds_income_groups=ds_income_groups,
+        min_num_values_per_year=1,
+    )
+    tb_reg = tb_reg[tb_reg["country"].isin(REGIONS)]
+    tb = pr.concat([tb, tb_reg], axis=0)
+    # Add correction factor to estimate polio cases based on reported cases.
+    tb = add_correction_factor(tb)
+    tb["estimated_cases"] = tb["total_cases"] * tb["correction_factor"]
+    # Add polio surveillance status based on the screening and testing rates.
+    tb = add_screening_and_testing(tb, tb_regions, ds_regions)
+    tb = add_cases_per_million(tb, tb_population)
+    tb = tb.set_index(["country", "year"], verify_integrity=True)
+    tb.metadata.short_name = "polio"
```

Comment:
Instead of the last two lines, you can use `tb.format(short_name="polio")` (and optionally `sort_rows=False` if you don't want to sort).